pkg/TIPA.py

```
@@ -393,28 +406,28 @@ def determine_inventory(self, diff_num: int) -> bool:
         if self.debug_mode >= 1:
             logger.info("In raid screenshot")
         return False
-        
+
     def get_search_areas(self, inventory: bool) -> tuple:
         if inventory:
             return (
                 (self.mouse_pos["x"] - 16, self.mouse_pos["y"] - 42, self.mouse_pos["x"] + 420, self.mouse_pos["y"] - 10),
                 (self.mouse_pos["x"] - 400, self.mouse_pos["y"] - 65, self.mouse_pos["x"] + 420, self.mouse_pos["y"] - 10)
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)